pkg/descheduler/strategies/topologyspreadconstraint_test.go

```diff
@@ -1041,3 +1041,50 @@ func getDefaultTopologyConstraints(maxSkew int32) []v1.TopologySpreadConstraint
 		},
 	}
 }
+
+func TestCheckIdenticalConstraints(t *testing.T) {
+	newConstrainSame := v1.TopologySpreadConstraint{
+		MaxSkew:           2,
+		TopologyKey:       "zone",
+		WhenUnsatisfiable: v1.DoNotSchedule,
+		LabelSelector:     &metav1.LabelSelector{MatchLabels: map[string]string{"foo": "bar"}},
+	}
+	newConstrainDifferent := v1.TopologySpreadConstraint{
+		MaxSkew:           3,
+		TopologyKey:       "node",
+		WhenUnsatisfiable: v1.DoNotSchedule,
+		LabelSelector:     &metav1.LabelSelector{MatchLabels: map[string]string{"foo": "bar"}},
+	}
+	// Unique constrains, but with same content (except for different LabelSelector addresses)
+	uniqueConstrains := make(map[v1.TopologySpreadConstraint]struct{})
+	for i := 1; i < 5; i++ {
+		uniqueConstrains[v1.TopologySpreadConstraint{
+			MaxSkew:           2,
```

Comment:
Should the index `i` be used here? Or, as part of the `LabelSelector`? Otherwise, all constraints are identical.

Reply:
For some reason I wanted a bunch of identical Constrains (any number), but tbh we just need one, and try to add another one. I did that